project-base/app/src/DataFixtures/Demo/CategoryParameterDataFixture.php

```diff
@@ -5,13 +5,13 @@
 namespace App\DataFixtures\Demo;
 
 use App\Model\Category\Category;
-use App\Model\Product\Parameter\Parameter;
```

Comment:
this PR should also have the upgrade notes for storefront and backend. 

use `./phing upgrade-generate` to generate templates and take a look at the other upgrade notes for inspiration